tests/test_urls.py

```diff
@@ -27,6 +28,10 @@ def remove_sphinx_build_output():
             shutil.rmtree(build_path)
 
 
+@pytest.mark.sphinx(srcdir=srcdir)
+def test_parallel_build():
+    subprocess.check_call('sphinx-build -j 2 -W -b html tests/examples/parallel-build build', shell=True)
```

Comment:
can't we call `app.build` here instead, with a parallel argument here in some way?

Reply:
I mean, similar to what we are doing in the rest of the tests.

Reply:
The other tests rely on SphinxTestApp, which does not have a way to pass parallel=2 through to the Sphinx constructor. I submitted a PR to add it, but we would have to wait for the next sphinx release, and it would not be usable for your testing that runs on older versions of sphinx. You can see a discussion of the problem in a PR for another extension: https://github.com/executablebooks/sphinx-book-theme/pull/225#discussion_r497907074

Reply:
I see. Makes sense. So, should we check stderr or exit code or similar here to be sure that the subprocess didn't failed?

Reply:
Oh, check call_raises an exception.

Reply:
Right. And I added the test first and verified that it caught the problem before adding the fix.